deploy/charts/rook-ceph-cluster/values.yaml

```
@@ -47,6 +47,9 @@ monitoring:
   # Monitoring settings for external clusters:
   # externalMgrEndpoints: <list of endpoints>
   # externalMgrPrometheusPort: <port>
+  prometheusRule:
```

How about a simple comment on the new setting?
```suggestion
  # allow adding custom labels and annotations to the prometheus rule
  prometheusRule:
```

---

@travisn Done. Sorry for the delay.